src/MDMTransitionTarget.h

```diff
@@ -19,16 +19,19 @@
 #import <Foundation/Foundation.h>
 #import <UIKit/UIKit.h>
 
+#ifndef MDM_SUBCLASSING_RESTRICTED
 #if defined(__has_attribute) && __has_attribute(objc_subclassing_restricted)
-#define MDMSUBCLASSING_RESTRICTED __attribute__((objc_subclassing_restricted))
+#define MDM_SUBCLASSING_RESTRICTED __attribute__((objc_subclassing_restricted))
 #else
-#define MDMSUBCLASSING_RESTRICTED
+#define MDM_SUBCLASSING_RESTRICTED
 #endif
+#endif  // #ifndef MDM_SUBCLASSING_RESTRICTED
 
 /**
  A target for a view controller transition.
  */
 NS_SWIFT_NAME(TransitionTarget)
+MDM_SUBCLASSING_RESTRICTED
```

Comment:
If I understand correctly this may be a breaking change (and not a style update) for downstream clients, yes? They can no longer subclass `MDMTransitionTarget`?

Reply:
Yep - typically this would be a breaking change. In this case we haven't release a stable version of the library yet though so it won't impact our release number.

Reply:
SGTM. Would you please add a commit message body summarizing the "why" of these changes?

Reply:
Done.